examples/circuitplayground_acceleration_neopixels.py

```diff
@@ -13,15 +15,17 @@
         cpx.pixels.fill((0, 0, 0))
         continue
     else:
-        R = 0
-        G = 0
-        B = 0
-        x, y, z = cpx.acceleration
-        print((x, y, z))
-        if x:
-            R = R + abs(int(x))
-        if y:
-            G = G + abs(int(y))
-        if z:
-            B = B + abs(int(z))
-        cpx.pixels.fill((R, G, B))
+        def color_amount(accel_component):
```

Comment:
We typically include `def`s before the loop at the top of the code below the `import`s. Consider moving this.